launchable/commands/subset.py

```diff
@@ -178,24 +179,63 @@ def subset(
     lineage: Optional[str] = None,
     prioritize_tests_failed_within_hours: Optional[int] = None,
 ):
+    tracking_client = TrackingClient(Tracking.Command.SUBSET)
+
     if is_observation and is_get_tests_from_previous_sessions:
+        msg = "Cannot use --observation and --get-tests-from-previous-sessions options at the same time"
         click.echo(
             click.style(
-                "Cannot use --observation and --get-tests-from-previous-sessions options at the same time",
+                msg,
                 fg="red"),
             err=True,
         )
+        org, workspace = get_org_workspace()
+        tracking_client.send_error_event(
```

Comment:
it’s ok in another PR but how about getting and setting org/workspace value in send_error_event?

Reply:
Sure, I'll address it in the next PR.